### PR TITLE
Add a test banner if its running in development mode (TAM-150)

### PR DESCRIPTION
**Pull request by urshala**

### CODE REVIEW

File: respa_admin/templates/respa_admin/_nav.html [hunk 1]
```diff
@@ -59,7 +59,11 @@ <h3>Respa Admin</h3>
         </div><!-- /.navbar-collapse -->
     </div><!-- /.container-fluid -->
 </nav>
-
+<nav>
+  {%  if PRODUCTION_MODE == 'development' %}
+    <div role="alert" class="test-site-message alert alert-danger">Tämä on Respan testiversio</div>
```
Comment (wiltso) [inline respa_admin/templates/respa_admin/_nav.html]: Translated string?

File: respa/settings.py [hunk 2]
```diff
@@ -211,6 +213,8 @@ def get_git_revision_hash():
 TEST_RUNNER = 'respa.test_runner.PyTestShimRunner'
 TEST_PERFORMANCE = False
 
+PRODUCTION_MODE = env('PRODUCTION_MODE')
```
Comment (wiltso) [inline respa/settings.py:216]: Should this be based on the DEBUG setting instead?

Comment (urshala) [reply respa/settings.py:216]: The QA instance runs with DEBUG=False. So couldn't use that.

File: respa_admin/templates/respa_admin/_nav.html (hunk 1 above)
Comment (urshala) [reply respa_admin/templates/respa_admin/_nav.html]: Added!

**Review by wiltso — APPROVED** [review]
Nice work!

**Pull request merged**